lib/src/adapters/NativeEventsReceiver.ts

```diff
@@ -31,39 +33,57 @@ export class NativeEventsReceiver {
     return this.emitter.addListener('RNN.AppLaunched', callback);
   }
 
-  public registerComponentDidAppearListener(callback: (event: ComponentDidAppearEvent) => void): EventSubscription {
+  public registerComponentDidAppearListener(
```

Comment:
Lets return these method declarations to single line - I think it's readable enough in a single line.